src/powerquery-language-services/analysis.ts

```diff
@@ -71,64 +74,45 @@ abstract class AnalysisBase implements Analysis {
         // - only include current query name after @
         // - don't return completion items when on lefthand side of assignment
```

Comment:
I think we can remove all TODOs except (1).

Reply:
I'll clean this up in a subsequent change